app/styles/components/reports/subject.scss

```diff
@@ -1,28 +1,47 @@
-@use "../../ilios-common/mixins" as cm;
-@use "../../mixins" as m;
+@use "../../ilios-common/mixins" as m;
```

Comment:
I aliased all the common mixins as `cm` to make it a bit easier to see, while in the sass where something was coming from without having to scroll up. I don't see any way to code lint this, so it's just a convention at this point, but I think it's a good one.

Reply:
:heavy_check_mark: